openedx/features/edly/permissions.py

```diff
@@ -0,0 +1,15 @@
+"""
+Custom permissions for edly.
+"""
+from rest_framework.permissions import BasePermission
+
+
+class CanAccessEdxAPI(BasePermission):
+    """
+    Checks if a user ca access Edx API.
+    """
+
+    def has_permission(self, request, view):
+        return request.user.is_staff or request.user.edly_profile.edly_sub_organizations.filter(
+            lms_site=request.site
+        ).exists()
```

Comment:
for second clause can we write it like this?
```request.site.edly_sub_org_for_lms.slug in request.user.edly_profile.get_linked_edly_sub_organizations```

Reply:
Done